ocp_cleanup.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster || true
-
+sudo virsh destroy "${CLUSTER_NAME}-bootstrap"
+sudo virsh undefine "${CLUSTER_NAME}-bootstrap" --remove-all-storage
+sudo virsh vol-delete "${CLUSTER_NAME}-bootstrap.ign" --pool default
```

Comment:
I think s/default/images/ ?

Reply:
Hmm, I think the bootstrap storage uses the default pool?

```
[shardy@tripleodev2 dev-scripts]$ sudo virsh pool-list
 Name                 State      Autostart 
-------------------------------------------
 default              active     yes       
 oooq_pool            active     yes       

[shardy@tripleodev2 dev-scripts]$ sudo virsh vol-list --pool default
 Name                 Path                                    
------------------------------------------------------------------------------
 ostest-bootstrap.ign /var/lib/libvirt/images/ostest-bootstrap.ign
 ostest-bootstrap.qcow2 /var/lib/libvirt/images/ostest-bootstrap.qcow2
```

Are you seeing something different in your environment?

Reply:
I found that the --remove-all-storage removes the ostest-bootstrap.qcow2  but I had to manually clean the ign volume.

Reply:
Weird, my pool name is different

[root@host11 dev-scripts]# sudo virsh pool-list
 Name                 State      Autostart 
-------------------------------------------
 images               active     yes       
 oooq_pool            active     yes       

[root@host11 dev-scripts]# sudo virsh vol-list --pool default
error: failed to get pool 'default'
error: Storage pool not found: no storage pool with matching name 'default'

[root@host11 dev-scripts]# sudo virsh vol-list --pool images
 Name                 Path                                    
------------------------------------------------------------------------------
 ostest-bootstrap.ign /var/lib/libvirt/images/ostest-bootstrap.ign
 ostest-bootstrap.qcow2 /var/lib/libvirt/images/ostest-bootstrap.qcow2

I'll see if I can figure out why later

Reply:
That's weird, I'll mark this WIP again until we figure out what's going on